packages/terra-popup/src/Popup.jsx

```diff
@@ -76,7 +76,7 @@ const propTypes = {
    */
   isArrowDisplayed: PropTypes.bool,
   /**
-   * Should the popup content have tab focus. Set this is your content doesn't contain any focusable elements.
+   * Should the popup content have tab focus. Set this to `true` if your content doesn't contain any focusable elements.
```

Comment:
I also think I originally documented this incorrectly. Setting the value to true should be done when you want to disable the popups default tab focus. Meaning you "do" have focusable elements and want to take focus away from the default popup frame.

Reply:
Using that prop may actually fix the screen reader issue by itself.

Reply:
Did a quick test with VO on iOS and setting the prop to true allows the virutal focus indicator to shift correctly. I'll work on the wording to make this more clear what the prop does and update our popup examples that have focusable content within them to use this prop.